src/Forms/GridField/GridFieldDetailForm_ItemRequest.php

```diff
@@ -410,6 +468,94 @@ public function doSave($data, $form)
         return $this->redirectAfterSave($isNewRecord);
     }
 
+    /**
+     * Goes to the next record
+     * @param  array $data The form data
+     * @param  Form $form The Form object
+     * @return HTTPResponse
+     */
+    public function doNext($data, $form)
+    {
+        Controller::curr()->getResponse()->addHeader("X-Pjax", "Content");
+        $link = $this->getEditLink($this->getNextRecordID());
+        return Controller::curr()->redirect($link);
```

Comment:
Shouldn't need `Controller::curr()` here as we're in a controller right now. `$this` should work fine, I would think?